app/manage/permissions/delete-team.js

```diff
@@ -0,0 +1,5 @@
+const { isModerator } = require('../../lib/team')
+
+module.exports = async function ({ user_id }, { params: { id } }) {
```

Comment:
Looking at these permission functions it wasn't clear to me at first that these are middleware functions. It would be useful to add jsdoc comments describing that the arguments are `res.locals` and `req`, which properties are used to perform the check, and a one-liner description of what we're checking for (like checking if the user is a moderator of the team, in this case).